contrib/opencensus-ext-ocagent/opencensus/ext/ocagent/utils/__init__.py

```diff
@@ -0,0 +1,65 @@
+import datetime
+import os
+import socket
+
+from google.protobuf.internal.well_known_types import ParseError
+from google.protobuf.timestamp_pb2 import Timestamp
+from opencensus.common.version import __version__
```

Comment:
Minor suggestion, give it an explicit name (e.g. opencensus_version) so it won't be confused with extension version.

Reply:
Done